cli/cli.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/hasura/graphql-engine/cli/telemetry"
 	"github.com/hasura/graphql-engine/cli/util"
 	"github.com/hasura/graphql-engine/cli/version"
+	"github.com/joho/godotenv"
```

Comment:
@ShraddhaAg @shahidhk  Instead of using godotenv, can we use https://github.com/subosito/gotenv (we already have this dependency due to viper).

Reply:
Replaced the dependency as suggested.